server/src/test/java/org/apache/druid/query/lookup/LookupReferencesManagerTest.java

```diff
@@ -38,19 +40,28 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
 
 import java.io.IOException;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
+@RunWith(JUnitParamsRunner.class)
 public class LookupReferencesManagerTest
 {
   private static final String LOOKUP_TIER = "lookupTier";
+
+  // null indicates that all lookups need to be loaded.
```

Comment:
Nit: instead of this comment, you may just name the field that indicates this. e.g. `ALL_LOOKUPS`.

Reply:
Do you mean renaming the constant? `LOOKUPS_TO_LOAD = null` -> `ALL_LOOKUPS = null`?

Reply:
I've renamed the constant and removed the comment. Please let me know if this wasn't the suggestion. Thanks!